src/value.c

```diff
@@ -216,6 +225,10 @@ int SIValue_ToString(SIValue v, char *buf, size_t len) {
   case T_NEGINF:
     bytes_written = snprintf(buf, len, "-inf");
     break;
+  case T_NODE:
+  case T_EDGE:
+    bytes_written = snprintf(buf, len, "%lu", ENTITY_GET_ID((GraphEntity*)v.ptrval));
```

Comment:
Is that the string representation of a graph entity we want to use?

Reply:
As a user-facing representation, I agree that this is unsatisfactory. This logic also gets called for things like making group keys for op_aggregate and DISTINCT for result sets, though, so I want it to be as compact as possible.

Reply:
👍